volumina/pixelpipeline/imagepump.py

```diff
@@ -351,15 +351,15 @@ def __init__(self, layerStackModel, sliceProjection, sync_along=(0, 1, 2)):
 
     # mappings
     def layerToSliceSources(self, layer):
```

Comment:
don't know if this should be renamed to be uber consistent

Reply:
renamed `layerToPlanarSliceSources`